### DISCUSSION TITLE
为什么我的bot总是initializing两次

### DISCUSSION THREAD

**Discussion by yhzcake** [nonebot/discussions#374] [问答]
如题
![image](https://github.com/nonebot/discussions/assets/80627671/804b3aab-5bfd-403b-8733-3fc4db764c74)
生成了bot.py
去掉就变成这样了
![image](https://github.com/nonebot/discussions/assets/80627671/896ada65-e27c-4f6b-be6d-af8eb7b8ebc7)
按照在其他issue里的说法好像是这个的问题nonebot.run(app="__mp_main__:app")
有解决办法嘛

**Comment by yanyongyu** [comment]
删除你的`fastapi_reload`配置项
